amplify/src/main/java/com/github/stkent/amplify/tracking/ApplicationInfoProvider.java

```diff
@@ -35,6 +37,23 @@ public ApplicationInfoProvider(@NonNull final Context context) {
 
     @NonNull
     @Override
+    public String getDeviceName() {
+        String manufacturer = Build.MANUFACTURER;
+        String model = Build.MODEL;
+
+        String deviceName;
+
+        if (model.startsWith(manufacturer)) {
+            deviceName = StringUtils.capitalize(model);
+        } else {
+            deviceName = StringUtils.capitalize(manufacturer) + " " + model;
+        }
```

Comment:
No idea! Might be good to log all these properties in JJs real quick, and see what each outputs on a couple devices?